### ISSUE TITLE
Windows - PathnameExtensions#file issue - File.open

### ISSUE THREAD

**Issue by MSP-Greg** [bhb/test_construct#7]
While working on another gem that uses this gem, I came across an issue with Windows testing.

The following line leaves encoding and line endings such that the tests broke.

https://github.com/bhb/test_construct/blob/944e5b028256bae8173765e53579ec4e8ac2bdd7/lib/test_construct/pathname_extensions.rb#L20

I replaced it with:
```ruby
      mode = RUBY_PLATFORM =~ /mingw|mswin/ ? 'wb:UTF-8' : 'w'
      File.open(path, mode) do |f|
```

and problem solved.  I could do a PR...

**Comment by bhb** [comment]
@MSP-Greg Thanks for reporting this! Yes, a PR would be great, thanks!

**Comment by MSP-Greg** [comment]
See https://github.com/bhb/test_construct/pull/8

**Comment by bhb** [comment]
Resolved with https://github.com/bhb/test_construct/pull/8

**Comment by bhb** [comment]
Just released 2.0.2 which includes this change. Thanks!
